doc/backend_vars.asciidoc

```diff
@@ -25,6 +25,7 @@ TIMEOUT_SCALE;integer;1;This scale parameter can be used based on performance of
 PAUSE_AT;string;;Test module (name or fullname) to pause test execution at. To be used together with the openQA developer mode to continue test execution. Be aware that the openQA web UI will only reflect this within the developer mode after confirming to control the test.
 PAUSE_ON_SCREEN_MISMATCH;boolean;0;Pause test execution on the next screen mismatch. Same notes as for `PAUSE_AT` apply.
 PAUSE_ON_NEXT_COMMAND;boolean;0;Pause test execution on the next test API command. Same notes as for `PAUSE_AT` apply.
+_QUIET_SCRIPT_CALLS;boolean;0;Add quiet flag to all the calls to script_run and script_output.
```

Comment:
as far as I can see _QUIET_SCRIPT_CALLS has influence only for case `testapi::is_serial_terminal` returning true , might be usefull to mention this in variable description

Reply:
From my perspective, `is_serial_terminal` doesn't belong to the `quiet` switch. 
The `quiet` switch avoid the call of `record_serialresult()` in `wait_serial()` and validate_script_output()`. And this apply to  all kinds of consoles.

Reply:
@jlausuch have you tried to run it with some VNC test  case to see if it really make some difference and not produce any errors ?

Reply:
can you please also explain here what the benefit of the flag is? Stating that it just adds a flag is certainly correct but also a bit boring, don't you think? ;)

Reply:
done